src/cljs/org/broadinstitute/firecloud_ui/page/workspaces.cljs

```diff
@@ -217,16 +222,18 @@
 (defn- render-workspace-summary [workspace]
   [:div {:style {:margin "45px 25px"}}
    [:div {:style {:position "relative" :float "left" :display "inline-block"
-                  :top 0 :left 0 :width 290 :marginRight 40 :height "100%"}}
-    [:div {:style {:borderRadius 5 :backgroundColor (:success-green style/colors) :color "#fff"
-                   :fontSize "125%" :fontWeight 400 :padding 25 :textAlign "center"}}
-     [:span {:style {:display "inline-block" :verticalAlign "middle"
-                     :width "2.5em" :height "2.5em" :position "relative"
-                     :backgroundColor "#fff" :borderRadius "100%"}}
-      [:span {:style {:fontFamily "fontIcons" :color (:success-green style/colors) :fontSize "90%"
-                      :position "absolute"
-                      :top "50%" :left "50%" :transform "translate(-50%, -50%)"}} ""]]
-     [:span {:style {:marginLeft "1.5ex"}} "Complete"]]
+                  :top 0 :left 0 :width 290 :marginRight 40}}
```

Comment:
Might be worth adding a TODO here for making this a dynamic width so it takes up less space on smaller screens.